appserver/monitoring-console/core/src/main/java/fish/payara/monitoring/admin/SetMonitoringConsoleConfigurationCommand.java

```diff
@@ -61,22 +66,50 @@
 import org.glassfish.hk2.api.PerLookup;
 import org.glassfish.hk2.api.ServiceLocator;
 import org.jvnet.hk2.annotations.Service;
+import org.jvnet.hk2.config.ConfigSupport;
+import org.jvnet.hk2.config.SingleConfigCode;
+import org.jvnet.hk2.config.TransactionFailure;
 
 import com.sun.enterprise.config.serverbeans.Domain;
 import com.sun.enterprise.util.SystemPropertyConstants;
 
+import fish.payara.monitoring.configuration.MonitoringConsoleConfiguration;
+
 @Service(name = "set-monitoring-console-configuration")
 @PerLookup
 @ExecuteOn({RuntimeType.DAS})
 @TargetType({DAS})
+@RestEndpoints({
+    @RestEndpoint(configBean = Domain.class,
+            opType = RestEndpoint.OpType.POST,
+            path = "set-monitoring-console-configuration",
+            description = "Set Monitoring Console Configuration")
+})
 public class SetMonitoringConsoleConfigurationCommand implements AdminCommand {
 
+    private static final Logger LOGGER = Logger.getLogger(SetMonitoringConsoleConfigurationCommand.class.getName());
+
     private static final String MONITORING_CONSOLE_APP_NAME = "__monitoringconsole";
     private final static String GLASSFISH_LIB_INSTALL_APPLICATIONS = "glassfish/lib/install/applications";
 
     @Param(optional = true)
     private Boolean enabled;
 
+    @Param(optional = true, alias = "disable-watch")
+    private String disableWatch;
+
+    @Param(optional = true, alias = "enable-watch")
+    private String enableWatch;
+
+    @Param(optional = true, alias = "add-watch-name")
+    private String addWatchName;
+
+    @Param(optional = true, alias = "add-watch-json")
+    private String addWatchJson;
+
+    @Param(optional = true, alias = "remove-watch")
+    private String removeWatch;
+
```

Comment:
NB. I opted for a parameter for each action. This could also be done by adding a general parameter `watch` for the name and `data` for the JSON and another parameter `action` or so. But I found this easier to use. WDYT?